templates/core/.env.sample

```diff
@@ -20,7 +20,10 @@ SWAGGER_UI_CLIENT_ID=__CHANGE_ME__
 # TEST_ACCOUNT_CLIENT_SECRET=__CHANGE_ME__
 # TRE_URL=https://${TRE_ID}.westeurope.cloudapp.azure.com
 
-
+# If you are behind a firewall and you cannot browse
+# to https://ipecho.net/plain as its blocked then set
+# your local IP address here
```

Comment:
Should we be more explicit that "this is the IP address of the machine performing the deployment. e.g. your build agent, or your local machine"?

Reply:
i.e. spell it out why you might want to do this, and what this IP address represents